### PR TITLE
rewrite caregiver test in cypress

### PR DESCRIPTION
**Pull request by CBonade**
## Description
Rewrite of the caregivers test from Nightwatch to Cypress.  This is a very simple test that would likely be covered by our daily accessibility test, once it's converted from Nightwatch.

https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/25704

## Testing done
Makes it to the end of the test in Cypress Chrome test runner.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/tests/00.caregivers.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import environments from 'site/constants/environments';
+
+describe('Caregivers test', () => {
+  it('Loads the page body', () => {
+    if (
+      !(
+        manifest.e2eTestsDisabled &&
+        process.env.BUILDTYPE !== environments.LOCALHOST
+      )
+    )
```
Comment (ddzz) [inline src/applications/caregivers/tests/00.caregivers.cypress.spec.js:12]: Can we add a comment that explains this block?

Comment (CBonade) [reply src/applications/caregivers/tests/00.caregivers.cypress.spec.js]: Added!

**Review by dat-gcio — APPROVED** [review]
Thanks for doing this!

**Comment by pjhill** [comment]
Resolves [25704](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25704)

**Pull request merged**